xs2a-adapter-aspsp-registry/src/main/java/de/adorsys/xs2a/adapter/registry/AspspSearchServiceImpl.java

```diff
@@ -32,7 +32,9 @@ public AspspSearchServiceImpl(AspspRepository luceneAspspRepository) {
             throw new IbanException(e.getMessage());
         }
         if (bankCode == null) {
-            throw new IbanException("Failed to extract the bank code from the iban");
+            String errorMessage = "Failed to extract the bank code from the iban=" + iban;
```

Comment:
https://stackoverflow.com/questions/6639963/why-is-log-and-throw-considered-an-anti-pattern

Reply:
thanks, fixed